src/nextlabs_sdk/_cloudaz/_engine/_request_plan.py

```
@@ -9,5 +9,5 @@ class RequestPlan:
 
     method: str
     path: str
-    params: dict[str, int | str] | None = None  # noqa: WPS110
+    params: dict[str, int | str | bool] | None = None  # noqa: WPS110
```

Ruff (0.15.21)

[warning] Invalid rule code in `# noqa`: WPS110

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)